packages/console/run-tests.sh

```
@@ -0,0 +1,75 @@
+# compile it here when it is missing. It is a local, offline compile of KB of
+# JSON — a couple of seconds, and a no-op once present.
+if [ ! -f "src/paraglide/messages.js" ]; then
+  echo "Compiling paraglide messages (src/paraglide is generated and gitignored)"
+  npx paraglide-js compile --project ./project.inlang --outdir ./src/paraglide
+fi
+
+# Define the pattern to match your test files
+pattern="src/*.test.ts"
+
+# Expand the pattern into an array of files
+files=($(find src -type f -name "*.test.ts"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)